Shopify/cli
# `app config push` and `app update-url` commands dropped

### WHY are these changes introduced?
`versioned app configuration` has been 100% rolled out altogether with the `3.55.0` release. This means that the commands 
- `app config push`
- `app update-urls`

are not going to work anymore, not only for the `3.55.0` but for any other older version too

Those commands should be dropped from the CLI



### WHAT is this pull request doing?
- Removes the `commands` and `services`
- Removes APIs only used by these commands/services
- Removes any utility method only used by these commands/services
- Display command not found error if force flag is present instead of suggesting an alternative command


### How to test your changes?
- Run `dev` command, modify the scopes and it should work as expected
- Run `push` and `update-url` command and the agent should suggest a different command because those don't exist
- Run the commands with the `--force/-f` flag and the `not found command error` should be displayed


### Post-release steps



### Measuring impact

How do we know this change was effective? Please choose one:


### Checklist
